packages/gestalt/src/Dropdown.jsdom.test.js

```diff
@@ -397,60 +381,57 @@ describe('Dropdown', () => {
     fireEvent.keyDown(window.document, {
       keyCode: DOWN_ARROW,
     });
-
     expect(document.activeElement).toHaveAttribute('id', 'ex-7-item-1');
 
     fireEvent.keyDown(window.document, {
       keyCode: ENTER,
     });
-
-    expect(handleSelectMock).toHaveBeenCalledTimes(1);
+    expect(onSelectMock).toHaveBeenCalledTimes(1);
   });
 
-  it('should call link handleSelect when enter key is pressed', () => {
+  it('should call link onClick when enter key is pressed', () => {
```

Comment:
This is the test that's failing. I have no idea why. It doesn't register that mockOnClick was called.